<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="2" total="3" passed="0">
  <reporter-output>
    <line>
      <![CDATA[About to begin executing Suite Suite]]>
    </line>
    <line>
      <![CDATA[About to begin executing Test FirefoxTest]]>
    </line>
    <line>
      <![CDATA[About to begin executing Test Chore tests]]>
    </line>
    <line>
      <![CDATA[About to begin executing Test IETest]]>
    </line>
    <line>
      <![CDATA[About to begin executing following method : Cross.beforeTest]]>
    </line>
    <line>
      <![CDATA[About to begin executing following method : Cross.beforeTest]]>
    </line>
    <line>
      <![CDATA[About to begin executing following method : Cross.beforeTest]]>
    </line>
    <line>
      <![CDATA[Completed executing following method : Cross.beforeTest]]>
    </line>
    <line>
      <![CDATA[About to begin executing following method : Cross.spicejet]]>
    </line>
    <line>
      <![CDATA[Completed executing following method : Cross.spicejet]]>
    </line>
    <line>
      <![CDATA[Test Method resides in sample.Cross]]>
    </line>
    <line>
      <![CDATA[Test Status: Failed]]>
    </line>
    <line>
      <![CDATA[About to begin executing following method : Cross.afterTest]]>
    </line>
    <line>
      <![CDATA[Completed executing following method : Cross.afterTest]]>
    </line>
    <line>
      <![CDATA[About to begin executing Suite Suite]]>
    </line>
    <line>
      <![CDATA[About to begin executing Test FirefoxTest]]>
    </line>
    <line>
      <![CDATA[About to begin executing Test Chore tests]]>
    </line>
    <line>
      <![CDATA[About to begin executing Test IETest]]>
    </line>
    <line>
      <![CDATA[About to begin executing following method : Cross.beforeTest]]>
    </line>
    <line>
      <![CDATA[About to begin executing following method : Cross.beforeTest]]>
    </line>
    <line>
      <![CDATA[About to begin executing following method : Cross.beforeTest]]>
    </line>
    <line>
      <![CDATA[Completed executing following method : Cross.beforeTest]]>
    </line>
    <line>
      <![CDATA[About to begin executing Suite Suite]]>
    </line>
    <line>
      <![CDATA[About to begin executing Test FirefoxTest]]>
    </line>
    <line>
      <![CDATA[About to begin executing Test Chore tests]]>
    </line>
    <line>
      <![CDATA[About to begin executing Test IETest]]>
    </line>
    <line>
      <![CDATA[About to begin executing following method : Cross.beforeTest]]>
    </line>
    <line>
      <![CDATA[About to begin executing following method : Cross.beforeTest]]>
    </line>
    <line>
      <![CDATA[About to begin executing following method : Cross.beforeTest]]>
    </line>
    <line>
      <![CDATA[Completed executing following method : Cross.beforeTest]]>
    </line>
    <line>
      <![CDATA[About to begin executing following method : Cross.spicejet]]>
    </line>
    <line>
      <![CDATA[Completed executing following method : Cross.spicejet]]>
    </line>
    <line>
      <![CDATA[Test Method resides in sample.Cross]]>
    </line>
    <line>
      <![CDATA[Test Status: Failed]]>
    </line>
    <line>
      <![CDATA[About to begin executing following method : Cross.afterTest]]>
    </line>
    <line>
      <![CDATA[Completed executing following method : Cross.afterTest]]>
    </line>
  </reporter-output>
  <suite name="Suite" duration-ms="74977" started-at="2018-07-26T07:49:42Z" finished-at="2018-07-26T07:50:57Z">
    <groups>
    </groups>
    <test name="IETest" duration-ms="36267" started-at="2018-07-26T07:49:42Z" finished-at="2018-07-26T07:50:19Z">
      <class name="sample.Cross">
        <test-method status="PASS" signature="beforeTest(java.lang.String)[pri:0, instance:sample.Cross@366e2eef]" name="beforeTest" is-config="true" duration-ms="36234" started-at="2018-07-26T13:19:42Z" finished-at="2018-07-26T13:20:19Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[ie]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[Completed executing following method : Cross.beforeTest]]>
            </line>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="FAIL" signature="spicejet()[pri:0, instance:sample.Cross@366e2eef]" name="spicejet" duration-ms="8" started-at="2018-07-26T13:20:19Z" finished-at="2018-07-26T13:20:19Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at sample.Cross.spicejet(Cross.java:59)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:393)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
            <line>
              <![CDATA[Completed executing following method : Cross.spicejet]]>
            </line>
            <line>
              <![CDATA[Test Method resides in sample.Cross]]>
            </line>
            <line>
              <![CDATA[Test Status: Failed]]>
            </line>
          </reporter-output>
        </test-method> <!-- spicejet -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:sample.Cross@366e2eef]" name="afterTest" is-config="true" duration-ms="0" started-at="2018-07-26T13:20:19Z" finished-at="2018-07-26T13:20:19Z">
          <reporter-output>
            <line>
              <![CDATA[Completed executing following method : Cross.afterTest]]>
            </line>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- sample.Cross -->
    </test> <!-- IETest -->
    <test name="FirefoxTest" duration-ms="51977" started-at="2018-07-26T07:49:42Z" finished-at="2018-07-26T07:50:34Z">
      <class name="sample.Cross">
        <test-method status="FAIL" signature="beforeTest(java.lang.String)[pri:0, instance:sample.Cross@246ae04d]" name="beforeTest" is-config="true" duration-ms="0" started-at="2018-07-26T13:19:42Z" finished-at="2018-07-26T13:19:42Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[gecko]]>
              </value>
            </param>
          </params>
          <exception class="org.openqa.selenium.WebDriverException">
            <message>
              <![CDATA[Failed to connect to binary FirefoxBinary(C:\Program Files\Mozilla Firefox\firefox.exe) on port 7055; process output follows: 

Build info: version: '2.53.1', revision: 'a36b8b1cd5757287168e54b817830adce9b0158d', time: '2016-06-30 19:26:09'
System info: host: 'DESKTOP-T7322OL', ip: '192.168.43.165', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_121'
Driver info: driver.version: FirefoxDriver]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.WebDriverException: Failed to connect to binary FirefoxBinary(C:\Program Files\Mozilla Firefox\firefox.exe) on port 7055; process output follows: 

Build info: version: '2.53.1', revision: 'a36b8b1cd5757287168e54b817830adce9b0158d', time: '2016-06-30 19:26:09'
System info: host: 'DESKTOP-T7322OL', ip: '192.168.43.165', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_121'
Driver info: driver.version: FirefoxDriver
	at org.openqa.selenium.firefox.internal.NewProfileExtensionConnection.start(NewProfileExtensionConnection.java:125)
	at org.openqa.selenium.firefox.FirefoxDriver.startClient(FirefoxDriver.java:271)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:119)
	at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:216)
	at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:211)
	at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:207)
	at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:120)
	at sample.Cross.beforeTest(Cross.java:39)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:510)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:211)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.beforeRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:616)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:393)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.openqa.selenium.firefox.NotConnectedException: Unable to connect to host 127.0.0.1 on port 7055 after 45000 ms. Firefox console output:

	at org.openqa.selenium.firefox.internal.NewProfileExtensionConnection.start(NewProfileExtensionConnection.java:113)
	... 25 more
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.WebDriverException -->
          <reporter-output>
            <line>
              <![CDATA[Completed executing following method : Cross.beforeTest]]>
            </line>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="SKIP" signature="gmail()[pri:0, instance:sample.Cross@246ae04d]" name="gmail" duration-ms="1" started-at="2018-07-26T13:20:34Z" finished-at="2018-07-26T13:20:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- gmail -->
        <test-method status="SKIP" signature="afterTest()[pri:0, instance:sample.Cross@246ae04d]" name="afterTest" is-config="true" duration-ms="0" started-at="2018-07-26T13:20:34Z" finished-at="2018-07-26T13:20:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- sample.Cross -->
    </test> <!-- FirefoxTest -->
    <test name="Chore tests" duration-ms="74976" started-at="2018-07-26T07:49:42Z" finished-at="2018-07-26T07:50:57Z">
      <class name="sample.Cross">
        <test-method status="PASS" signature="beforeTest(java.lang.String)[pri:0, instance:sample.Cross@3a03464]" name="beforeTest" is-config="true" duration-ms="55085" started-at="2018-07-26T13:19:42Z" finished-at="2018-07-26T13:20:37Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[Completed executing following method : Cross.beforeTest]]>
            </line>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="FAIL" signature="spicejet()[pri:0, instance:sample.Cross@3a03464]" name="spicejet" duration-ms="19884" started-at="2018-07-26T13:20:37Z" finished-at="2018-07-26T13:20:57Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at sample.Cross.spicejet(Cross.java:65)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:393)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
            <line>
              <![CDATA[Completed executing following method : Cross.spicejet]]>
            </line>
            <line>
              <![CDATA[Test Method resides in sample.Cross]]>
            </line>
            <line>
              <![CDATA[Test Status: Failed]]>
            </line>
          </reporter-output>
        </test-method> <!-- spicejet -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:sample.Cross@3a03464]" name="afterTest" is-config="true" duration-ms="0" started-at="2018-07-26T13:20:57Z" finished-at="2018-07-26T13:20:57Z">
          <reporter-output>
            <line>
              <![CDATA[Completed executing following method : Cross.afterTest]]>
            </line>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- sample.Cross -->
    </test> <!-- Chore tests -->
  </suite> <!-- Suite -->
</testng-results>
